### PR TITLE
Join warnings advisories

### PR DESCRIPTION
**Pull request by ryancreatescopy**
There's a few places where we need to provide better information around Optimistic Registration

- Join landing page should display a warning if we're approaching the MACI cap as there's a chance a very worthy project might not make it into the round. The feeling is this warning should display when the number of projects in the challenge period + projects in the round is near the MACI cap (TBC). 
  - Because of this I think we should have a page that explains some of the MACI limitations (and benefits of course), I've set this up but haven't actually written the content.
- The Success page needs to be rewritten to be more like "Project submitted" as there is still a chance it won't get added. It essentially goes into a pending state. However, because there is a challenge period we can be explicit about how long a user will have to wait to hear more (i.e. a rejection) or see their project live (after challenge period). We will be able to email successful projects too hopefully to confirm this.

**TODOs**
- [x]  Add actual variables, at the moment content has placeholders, e.g. x days
- [x]  Fix MACI tooltip in warning message on Join page
- [ ] Write MACI content

**Outstanding questions**
- What are we doing with the security deposit for unsuccessful projects?
- Do we need a similar warning on the projectAdded page if we're near the MACI cap (to reiterate this?)

### CODE REVIEW

**Comment by samajammin** [comment]
@ryancreatescopy would you like some help with the lingering todos here? Are any of those outstanding questions still blocking you here?

**Comment by ryancreatescopy** [comment]
Yes, help needed.

File: vue-app/src/views/ProjectAdded.vue [hunk 1]
```diff
@@ -7,13 +7,16 @@
         <img src="@/assets/newrings.png" width="100%" />
         <div class="content">
           <span class="emoji">🎉</span>
-          <h1>Project added!</h1>
-          <div id="subtitle" class="subtitle">You’re on board this funding round!</div>
-          <p>Your project will be added to the next round and be visible in a few minutes. Check back soon to see your live project.</p>
-          <div class="contact">
+          <h1>Project submitted!</h1>
```
Comment (samajammin) [inline vue-app/src/views/ProjectAdded.vue]: Defer to more recent changes on this page.

**Comment by samajammin** [comment]
Pushed some updates but am marking this as "blocked". From what I can gather, there's currently no way to query the # of recipients already in the registry contract, which we need in order to calculate if the round is full or not.

I think there’s a couple approaches here:

1) Integrate with the subgraph that @daodesigner is building to query # of recipients in the registry.
2) Add a view function to the smart contract to return the length of recipients. This may require us to loop through the `recipients` mapping on the `BaseRecipientRegistry` smart contract & check if `recipient.removedAt == 0`.

**Pull request merged**